commit 2a0b139283935f01faea6e24d8def3c844f8fe1d
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Tue Oct 5 09:35:29 2010 -0700

    Staging: brcm80211: remove UNUSED_PARAMETER macro
    
    It's not needed anywhere.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/typedefs.h
+++ b/drivers/staging/brcm80211/include/typedefs.h
@@ -202,9 +202,6 @@ typedef float64 float_t;
 #undef TYPEDEF_FLOAT64
 #undef TYPEDEF_FLOAT_T
 
-/* Suppress unused parameter warning */
-#define UNUSED_PARAMETER(x) (void)(x)
-
 /*
  * Including the bcmdefs.h here, to make sure everyone including typedefs.h
  * gets this automatically
